tracer/build/_build/Build.VariableGenerations.cs

```diff
@@ -32,6 +32,7 @@ Target GenerateVariables
 
             void GenerateConditionVariables()
             {
+                GenerateConditionVariableBasedOnGitChange("isAppSecChanged", new[] { "tracer/src/Datadog.Trace/Iast", "tracer/src/Datadog.Tracer.Native/iast", "tracer/src/Datadog.Trace/AppSec" }, new string[] { });
```

Comment:
Just FYI, Appsec appears in other places too, e.g. ASP.NET/Core stuff, so this is a conservative list. I think that's fine, especially if you also add the "force" case above